src/ImageManipulation.php

```diff
@@ -782,6 +789,31 @@ public function getHeight()
         return 0;
     }
 
+    /**
+     * Return whether image will be lazy loaded
+     *
+     * @return bool
+     */
+    public function IsLazyLoaded()
+    {
+        if (!Config::inst()->get('SilverStripe\\Assets\\Image', 'lazy_loading_enabled')) {
+            return false;
+        }
+        return $this->lazyLoad;
+    }
+
+    /**
+     * Set whether image will be lazy loaded
+     *
+     * @param bool $lazyLoad
+     * @return $this
+     */
+    public function LazyLoad($lazyLoad)
```

Comment:
```suggestion
    public function setLazyLoad(bool $lazyLoad)
```

you'll also need a corredsponding `function getLazyLoad(): bool`